data/wanderer/wanderers middle.txt

```diff
@@ -3328,6 +3328,10 @@ event "wanderers: exiles have drones"
 		add variant 4
 			"Model 32"
 			"Model 16" 5
+		add variant
+			"Korath Ravager"
+		add variant
+			"Korath Ravager (Crippler)"
```

Comment:
Putting it here means these ships won't get added to the fleet for older saves.
Allowing older saves that have already had this event happen would require you to create an invisible mission that requires this event to have happened and triggers an event that adds just these variants to the fleet.

Reply:
Ah, I see. Do you think it's fine to do that so this is possible, with the invisible mission?

I will leave the Automata production part to you on your Moonbeam PR, since you're addressing that with the KIV's there too.

Reply:
I guess for the sake of this PR, I *won't* introduce it then, and leave that to Zitchas.